src/UI/Seller/src/app/orders/components/order-details/order-details.component.html

```diff
@@ -39,7 +29,7 @@ <h3 class="d-flex justify-content-start align-items-center mb-0">
   </div>
   <div class="d-block text-right mb-2">
     <span class="font-weight-bold">
-      Order Placed: {{ _order?.DateSubmitted | date: 'medium' }}
+      Order Placed: {{ getOrderDate() | date: 'medium' }}
```

Comment:
Not a deal breaker, but the order should now have xp.QuoteSubmittedDate

Reply:
Yep, this method returns that if it's a quote order, otherwise it returns DateSubmitted

Reply:
You are so above average!